# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dosfstools/linux-2.6.11.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dosfstools-2.9/mkdosfs/mkdosfs.c.vanilla	2005-03-05 20:10:43.795598840 +0100
+++ dosfstools-2.9/mkdosfs/mkdosfs.c	2005-03-05 20:11:25.327285064 +0100
@@ -65,6 +65,7 @@
 #include <unistd.h>
 #include <time.h>
 #include <errno.h>
+#include <asm/types.h>
 
 #if __BYTE_ORDER == __BIG_ENDIAN
 
